src/test/java/com/epam/university/java/core/task004/Task004Test.java

```diff
@@ -1,7 +1,11 @@
 package com.epam.university.java.core.task004;
 
 import com.epam.university.java.core.helper.TestHelper;
+//<<<<<<< HEAD
```

Comment:
This looks like merge conflict. You should resolve it before merge

Reply:
Ok, thank you for all answers, I'll fix it